src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -11,7 +11,7 @@
     public static final String MESSAGE_INVALID_TASK_DISPLAYED_INDEX = "The task index provided is invalid";
     public static final String MESSAGE_EVENTS_LISTED_OVERVIEW = "%1$d events listed!";
     public static final String MESSAGE_PAST_EVENT_END_DATE_TIME = "The event end date and end time provided are past";
-    public static final String MESSAGE_PAST_EVENT_DEADLINE = "The task deadline provided is past";
+    public static final String MESSAGE_PAST_DEADLINE = "The task deadline provided is past";
```

Comment:
can we rephrase the message to "Please ensure the deadline provided is not earlier than today. This means only today and date later than today are allowed." or something similar haha @SoulUseless a better suggestion? same for the error message for event because the original message sounds a bit wrong hmmm